src/alert/internal.tsx

```diff
@@ -42,7 +42,7 @@ export default function InternalAlert({
   ...rest
 }: InternalAlertProps) {
   const baseProps = getBaseProps(rest);
-  const format = useInternalI18n('alert');
+  const i18n = useInternalI18n('alert');
```

Comment:
Not sure about this change; doesn't `useInternalI18n` return `intl.formatMessage` or a similar function which will have to format the message (as in: find the translated version, interpolate values)?

Actually, would a name like `useFormatter` or `useI18nFormatter` make more sense for the hook?

Or am I misunderstanding how this works?